sample-app/Jenkinsfile

```diff
@@ -19,20 +19,12 @@ node {
   switch (env.BRANCH_NAME) {
     // Roll out to staging
     case "staging":
-        // Change deployed image in staging to the one we just built
-        sh("sed -i.bak 's#gcr.io/cloud-solutions-images/gceme:1.0.0#${imageTag}#' ./k8s/staging/*.yaml")
-        sh("kubectl --namespace=production apply -f k8s/services/")
-        sh("kubectl --namespace=production apply -f k8s/staging/")
-        sh("echo http://`kubectl --namespace=production get service/${feSvcName} --output=json | jq -r '.status.loadBalancer.ingress[0].ip'` > ${feSvcName}")
+        sh("kubectl --namespace=staging set image deployments/${appName} ${appName}=${imageTag}"
```

Comment:
@viglesiasce so are you saying this tutorial isn't following the staging/prod pattern anymore, but using a canary pattern? I was confused why the staging section of the Jenkins config was using the production namespace too...